### PR TITLE
709 dup import file headers

### PR DESCRIPTION
**Pull request by adrian-lara**
#### Any background context you want to provide?
SEED hangs when attempting to map columns for a file with duplicate file headers

#### What's this PR do?
Prevent mapping attempts on a file that has duplicate file headers.

#### How should this be manually tested?
Import mapping page disables the "Map Your Data" button and provides an updated warning if there are duplicate file headers.

Also, in case mapping for such a file is attempted via API - Python is set to throw an error.

#### What are the relevant tickets?
#709

### CODE REVIEW

File: locale/en_US/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -1668,8 +1668,8 @@ msgstr "Please go to the following page and setup your account:"
 msgid "Please join the SEED-Platform Dev forum where you can connect with other developers."
 msgstr "Please join the SEED-Platform Dev forum where you can connect with other developers."
 
-msgid "Please review SEED headers.  Duplicate values are not allowed."
-msgstr "Please review SEED headers.  Duplicate values are not allowed."
+msgid "Please review SEED headers or Data File headers. Duplicate values are not allowed for either."
```
Comment (nllong) [inline locale/en_US/LC_MESSAGES/django.po]: Can we reword to 'Please review SEED column names or the headers of the file being imported. Duplicate values are not allowed in either case.'

**Comment by adrian-lara** [comment]
Updated the alert text.

Also, I am seeing the warning persist even when the SEED column value is changed
Screen Shot 2019-12-12 at 9 59 01 AM

**Comment by nllong** [comment]
> Updated the alert text.
> 
> Also, I am seeing the warning persist even when the SEED column value is changed
>

Let me test again. It definitely wasn't updating.

**Comment by nllong** [comment]
Looks good. I'll merge down once CI passes.

**Pull request merged**